lib/screens/home_screen.dart

```diff
@@ -60,10 +121,45 @@ class HomeScreen extends StatelessWidget {
         ),
         body: Column(
           children: <Widget>[
-            Expanded(child: Map()),
+            Expanded(child: MapView()),
             Expanded(child: FriendList()),
             CurrentUser()
           ],
         ));
   }
 }
+
+class FirebaseUserConnector {
```

Comment:
Will there be other widgets (such as the `friendRequests` / `friendSearch`) that would use a FirebaseUserConnector? We might want to move it outside of this home_screen.dart file